Exiled.API/Features/Ragdoll.cs

```diff
@@ -0,0 +1,337 @@
+// -----------------------------------------------------------------------
+// <copyright file="Ragdoll.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features
+{
+    using System.Linq;
+
+    using UnityEngine;
+
+    /// <summary>
+    /// A set of tools to handle the ragdolls more easily.
+    /// </summary>
+    public class Ragdoll
+    {
+        private readonly global::Ragdoll ragdoll;
```

Comment:
Nice on all of the properties/methods, but if any new features are added and EXILED isn't changed, you'll have to either use reflection or getcomponents to access the underlying ragdoll class.

I believe that, as a rule, we should definitely try to abstract away most of the basegame interactions, but we should not prevent them. Keeping the underlying class private serves no real purpose, other than to make it harder to interact with basegame code, which we should NOT be doing. Every one of these wrapper classes should try its best to fully support what it's wrapping, but it shouldn't prevent plugins from accessing it.

Reply:
Could a `Player.Ragdoll`make sense? You'd be able to access it easily.

Reply:
Can't players have more than one Ragdoll? Also, I don't think it makes sense to put access to the ragdoll on the Player class instead of the Ragdoll class.

Reply:
That's true

Reply:
The classes reference to the base-game equivalent should be a public property, incase anyone ever needs to access it directly for something that the API alone cannot do.